src/lib.rs

```diff
@@ -348,6 +348,24 @@ impl Capture<Offline> {
             })
         }
     }
+
+    /// Opens an offline capture handle from a pcap dump file, given a path.
```

Comment:
This should specify the point of the second function so that people reading the documentation can understand the difference.